Application.mk

```diff
@@ -1,5 +1,6 @@
-APP_ABI := arm64-v8a
+APP_ABI := arm64-v8a  # armeabi-v7a  # has to match architecture of Beatsaber
```

Comment:
I feel like this might just be more confusing than anything else ? Are there still people writing mods for BS versions from last year ?

Reply:
LOL fair
I can't even remember which version the switch happened in